### PR TITLE
add config flags to db configure create

### PR DESCRIPTION
**Pull request by michellescripts**
supports #12713

Adds flags to `teleport db configure create`. The added flags are currently leveraged by Cloud when running `teleport db start`. Adding these to `configure` will allow us to transition our instructions away from `teleport db start`

[Database Access Configuration Reference](https://goteleport.com/docs/database-access/reference/configuration/)
[X] --aws-region
[X] --aws-redshift-cluster-id
[X] --ad-domain
[X] --ad-spn
[X] --gcp-project-id
[X] --gcp-instance-id
[X] --ad-keytab-file
[X] --ca-cert -> --ca-cert-file

### CODE REVIEW

File: tool/teleport/common/teleport.go [hunk 1]
```diff
@@ -238,6 +238,14 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
...
+	dbConfigureCreate.Flag("aws-redshift-cluster-id", "(Only for Redshift) Redshift database cluster identifier.").StringVar(&dbConfigCreateFlags.DatabaseAWSRedshiftClusterID)
+	dbConfigureCreate.Flag("ad-domain", "(Only for SQL Server) Active Directory domain.").StringVar(&dbConfigCreateFlags.DatabaseADDomain)
+	dbConfigureCreate.Flag("ad-spn", "(Only for SQL Server) Service Principal Name for Active Directory auth.").StringVar(&dbConfigCreateFlags.DatabaseADSPN)
+	dbConfigureCreate.Flag("ad-keytab-file", "(Only for SQL Server) Kerberos keytab file.").StringVar(&dbConfigCreateFlags.DatabaseADKeytabFile)
+	dbConfigureCreate.Flag("gcp-project-id", "(Only for Cloud SQL) GCP Cloud SQL project identifier.").StringVar(&dbConfigCreateFlags.DatabaseGCPProjectID)
+	dbConfigureCreate.Flag("gcp-instance-id", "(Only for Cloud SQL) GCP Cloud SQL instance identifier.").StringVar(&dbConfigCreateFlags.DatabaseGCPInstanceID)
+	dbConfigureCreate.Flag("ca-cert", "Database CA certificate path.").StringVar(&dbConfigCreateFlags.DatabaseCACertFile)
```
Comment (strideynet) [inline tool/teleport/common/teleport.go:248]: Slightly irrelevant to your PR, but it occurred to me whilst reviewing this: I do slightly worry that `ca-cert` is a little un-specific (given how much of Teleport tends to revolve around CAs and such) when not directly in the context of a Database config block in a config file, but seeing as this is the existing name for the flag in the `db start` command, there's probably not much we can do about that in this PR.

**Review by strideynet — APPROVED** [review]
Looks good to me.

File: tool/teleport/common/teleport.go (hunk 1 above)
Comment (r0mant) [inline tool/teleport/common/teleport.go:248]: Nit: Should we call this flag `ca-cert-file`? I know `teleport db start` uses `ca-cert` but that might have been an oversight, I think other CLI flags that accept file paths use `-file` suffix so wonder if we should do the same here for consistency.

**Review by r0mant — APPROVED** [review]
With a nit.

**Pull request merged**